src/vpc_connector.tf

```diff
@@ -0,0 +1,65 @@
+# vpc connectors
+# needed for Cloud Run / Cloud Functions to access Redis, Postgres, etc.
+# min $14 a month each
+
+# one per subnet
+# added to the subnet artifact to be shared with serverless apps in that region
+
+# This breaks if someone makes two subnet packages in the same region, in the same VPC.
+# That's pretty unlikely but this should be replaced with md-terraform-utils eventually.
+locals {
+  ranges = {
```

Comment:
I think this is a better user experience than having to specify the `/28` range. I can file an issue on this bundle for the bug:  A user can't make two subnets in the same VPC in the same region.